codalab/scripts/initialize_teams.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# Run this script from the CodaLab virtual environment to insert
+# initial data required by the web app into the database.
+
```

Comment:
This script allows to initialize teams information in already running Codalab instances. Is just a part of initialize.py.